docs/dev-concepts/start-building.md

```diff
@@ -128,6 +126,56 @@ Consider following these best practices when developing your app:
     - \# of active conversations: Conversations with at least one message
     - \# of returning conversations: Conversations with at least one message per participant
 
+## Performance best practices
+
+Follow these guidelines to optimize your app’s performance.
+
+### For JavaScript
+
+Using the JavaScript client SDK (`xmtp-js`):
+
+- [Upgrade to ≥v9.0.0](https://github.com/xmtp/xmtp-js/releases/tag/v9.0.0), which provides a 10x smaller web bundle size.
+- [Cache conversations](/docs/client-sdk/javascript/tutorials/quickstart#cache-conversations)
+- [Page through messages](/docs/client-sdk/javascript/tutorials/quickstart#list-messages-in-a-conversation-with-pagination)
+- [Compress messages](https://xmtp.org/docs/client-sdk/javascript/tutorials/quickstart#compression)
+- [Serialize decoded messages](https://github.com/xmtp/xmtp-js/releases/tag/v8.0.0)
+
+### For Kotlin
+
+Using the Kotlin client SDK (`xmtp-android`):
+
+- [Cache conversations](/docs/client-sdk/kotlin/tutorials/quickstart#cache-conversations)
+- [Page through messages](/docs/client-sdk/kotlin/tutorials/quickstart#list-messages-in-a-conversation-with-pagination)
+- [Compress messages](/docs/client-sdk/kotlin/tutorials/quickstart#compression)
+
+### For Swift
+
+Using the Swift client SDK (`xmtp-ios`):
+
+- [Page through messages](/docs/client-sdk/swift/tutorials/quickstart#list-messages-in-a-conversation-with-pagination)
+- [Compress messages](/docs/client-sdk/swift/tutorials/quickstart#compression)
+
+### For Dart
+
+Using the Dart client SDK (`xmtp-flutter`):
+
+- [Page through messages](/docs/client-sdk/dart/tutorials/quickstart#list-messages-in-a-conversation-with-pagination)
+- [Compress messages](/docs/client-sdk/dart/tutorials/quickstart#compression)
+
+### For React
+
+Usign the React client SDK (`react-sdk`):
+
+- [Page through messages](/docs/client-sdk/react/tutorials/quickstart#page-through-messages)
```

Comment:
Is it possible to "cache conversations," "compress messages," and "serialize decoded messages" with this SDK today? If yes, should we update the react-sdk README to provide sections and code examples for this functionality?